flight_planning/v1/telemetry.yaml

```diff
@@ -0,0 +1,250 @@
+info:
+  title: >-
+    Telemetry definitions used in flight planning interface.
+components:
+  schemas:
+    FlightTelemetry:
+      description: >-
+        When this information is present and the USS supports telemetry ingestion for a flight, the USS receiving this information should enqueue these telemetry reports to be delivered to the system, as if they were coming from the user, at (or soon after) the specified times.
+      type: object
+      properties:
+        states:
+          description: >-
+            The set of telemetry data that should be injected into the system (as if reported by the user or the user's system) at the appropriate times (and not before) for this flight.
+          type: array
+          items:
+            $ref: '#/components/schemas/AircraftState'
+          default: []
+    AircraftState:
+      description: State of an aircraft for the purposes of simulating the execution of a flight plan.
+      required:
+        - id
+        - timestamp
+      type: object
+      properties:
+        id:
+          description: >-
+            Unique identifier for this aircraft state/telemetry.  The content of an AircraftState with a given ID must not change over time.  Therefore, if a USS has already queued an aircraft state with this ID as telemetry to be delivered, this state may be ignored.
+          type: string
+        timestamp:
```

Comment:
Would there be any benefit to introducing a second timestamp to have both the time of measurement and the time of injection into the system? E.g. to simulate an UAS with connectivity issues sending its telemetry late.

Reply:
That does seem potentially useful if/where we want to verify behavior under delayed-transmission circumstances, but there are lots of things we could add and I think we should generally limit ourselves to the things we we need in order to perform tests we're actively planning.